### ISSUE TITLE
Notes from talking with Peter, Xiaojuan, Dan

### ISSUE THREAD

**Issue by bpbond** [PNNL-TES/rcq10#8]
Dan is working on a Q10-related study w/ ELM.

Peter asks:
- Does the Q10 separation reflect something intrinsic, or is [apparent](http://dx.doi.org/10.1038/nature04514) Q10? 
- How can we distinguish?
- This could occur with identical intrinsic Q10s if the substrate supply changes with temperature, for example (e.g. photosynthate). 
- Could calculate, using existing model outputs, the apparent Q10, right? (Intrinsic is 1.5 here we know.) That is, compute a Q10 based on the autotrophic respiration response to temperature, combining substrate and temperature effects
- In the data, look at annual versus perennial? Differences might reflect NSC / leaky root differences
- On a similar note, fertilization might exert an effect, i.e. with more N there's less excess C to be burned off / leaked out via roots?

**Comment by bpbond** [comment]
FYI @jinshijian @thorntonpe @dmricciuto ...Xiaojuan GitHub username?

<img width="693" alt="Screen Shot 2019-05-01 at 9 32 29 AM" src="https://user-images.githubusercontent.com/1956468/57019230-e1247a80-6bf3-11e9-9af4-fa4a00b66931.png">

<img width="683" alt="Screen Shot 2019-05-01 at 9 32 40 AM" src="https://user-images.githubusercontent.com/1956468/57019235-e4b80180-6bf3-11e9-9896-853cfd092156.png">

**Comment by dmricciuto** [comment]
Thanks for sharing these notes.  Xiaojuan's GitHub is xyk-ornl.

**Comment by bpbond** [comment]
>Could calculate, using existing model outputs, the apparent Q10, right? (Intrinsic is 1.5 here we know.) That is, compute a Q10 based on the autotrophic respiration response to temperature, combining substrate and temperature effects

Hi all, in particular Xiaojuan @xyk-ornl - @jinshijian and I are wondering, how much work would this be to do, from standard historical transient run? I.e., you guys extract soil surface CO2 flux (or whatever we need to calculate that), plus soil temperature at say 10 cm, for every timestep. We would then take these data and compute, for each day/night period, an "apparent Q10" (what _would_ have been measured at the soil surface by some simulated researcher in the model world) by fitting R=ae^bT to the data.

We know you guys are really busy right now, so no pressure, just wondering. Thanks!

**Comment by xyk-ornl** [comment]
Hi @bpbond  @jinshijian, it is pretty straightforward to output soil surface CO2 flux (soil respiration plus root respiration) and soil temperature for top 10cm at every time step. But doing that for the whole historical time period might be too much - taking long time to run and the output file would be huge (with half-hour time step). Maybe we can run only 1980-2010 and output these variables hourly - that would be more practical. Please let me know what you think. Thanks

**Comment by bpbond** [comment]
Oh gosh, agreed. I think 10 years (2001-2010) would be fine, outputted hourly. Would it be possible to add the photosynthesis->root C allocation flux too?

**Comment by xyk-ornl** [comment]
yes - we have that in the model and just need to add it to the hourly output list. Thanks

**Comment by bpbond** [comment]
So basically just a 10 (or even 5)-year uncoupled run outputting those variables...if not a huge amount of trouble (and absolutely at your convenience) that would be super. Thank you!

**Comment by xyk-ornl** [comment]
@bpbond @jinshijian

Just a heads up - After the CBGC paper and a few meeting and/or project related travels, I was back to working on this earlier this week. However I ran out of my NERSC allocation in the middle of the simulation - I was trying to run the transient simulation from 1850 to generate the 2000 initial condition. Since I will be out of office again starting next week, I've asked @dmricciuto to help us on this and hopefully we will have the outputs for you soon. I am sorry for the delay....

FYI, here are the variable list I put together

Root respiration
 
FROOT_MR
LIVECROOT_MR
CPOOL_FROOT_GR
CPOOL_FROOT_STORAGE_GR
TRANSFER_FROOT_GR
CPOOL_LIVECROOT_GR
CPOOL_LIVECROOT_STORAGE_GR
TRANSFER_LIVECROOT_GR
CPOOL_DEADCROOT_GR
CPOOL_DEADCROOT_STORAGE_GR
TRANSFER_DEADCROOT_GR
 
 
Soil respiration
 
HR
 
Root allocation from photosynthesis
 
CPOOL_TO_FROOTC
CPOOL_TO_FROOTC_STORAGE
CPOOL_TO_LIVECROOTC
CPOOL_TO_LIVECROOTC_STORAGE
CPOOL_TO_DEADCROOTC
CPOOL_TO_DEADCROOTC_STORAGE
 
Soil temperature for top 10cm soil
 
TSOI_10CM

**Comment by bpbond** [comment]
That looks great--thanks @xyk-ornl @dmricciuto

**Comment by dmricciuto** [comment]
@bpbond @jinshijian
I finished the runs on anvil but the outputs are huge - 1.6T.  Do you have an account there?  Or should I move to NERSC, or somewhere else?

**Comment by bpbond** [comment]
Uh wow, that's _large_. Yikes. @jinshijian we should talk to Sean to see if he has a multi-terabyte drive he could loan us.

I don't have an anvil account, but do on NERSC, so that would be easiest. Thanks Dan!

**Comment by jinshijian** [comment]
Wow, looks like you guys @xyk-ornl @dmricciuto have lots of progress on this！That is awesome, thank you so much. For some reason, my Gmail blocked the notification from Github, sorry for the delay response.

**Comment by dmricciuto** [comment]
Sorry I haven't gotten back to you with the output!  I finally transferred it over to NERSC, see here:
Please see:  /global/cscratch1/sd/ricciuto/e3sm_scratch/from_anvil  - this includes 10 years of hourly outputs for the variables listed above.
Let me know if you have any trouble accessing or if you need further explanation of the outputs.
